manifests/overlays/odhdashboardconfig/odh-dashboard-config.yaml

```diff
@@ -15,6 +15,7 @@ spec:
     enablement: true
     disableProjects: true
     disableModelServing: true
+    disableCustomServingRuntimes: true
```

Comment:
Moved the conversation into #1106 just for this, since this is still behind a feature flag we can either improve it or integrate the outcome of the other issue here.